### PR TITLE
Add V2 protocol support for BenchReadThroughputLatency

### PR DESCRIPTION
**Pull request by hangc0276**
### Motivation
The `BenchReadThroughputLatency` only supports the V3 protocol to read ledgers from the bookie server. Add the V2 protocol support.

### Changes
Add V2 protocol support for BenchReadThroughputLatency

### CODE REVIEW

File: bookkeeper-benchmark/src/main/java/org/apache/bookkeeper/benchmark/BenchReadThroughputLatency.java [hunk 1]
```diff
@@ -194,6 +196,10 @@ public static void main(String[] args) throws Exception {
         final ClientConfiguration conf = new ClientConfiguration();
         conf.setReadTimeout(sockTimeout).setZkServers(servers);
 
+        if (cmd.hasOption("useV2")) {
```
Comment (horizonzy) [inline bookkeeper-benchmark/src/main/java/org/apache/bookkeeper/benchmark/BenchReadThroughputLatency.java:199]: cmd.hasOption("useV2") -> cmd.getOptionValue("useV2")

Comment (horizonzy) [reply bookkeeper-benchmark/src/main/java/org/apache/bookkeeper/benchmark/BenchReadThroughputLatency.java:199]: > cmd.hasOption("useV2") -> cmd.getOptionValue("useV2")

Sorry, misunderstand it. It's right.

**Review by zymap — APPROVED** [review]

**Review by horizonzy — APPROVED** [review]
LGTM

**Review by wenbingshen — APPROVED** [review]
LGTM

**Pull request merged**